zgw/taak/src/main/kotlin/nl/nlportal/zgw/taak/autoconfigure/TaakConfig.kt

```diff
@@ -23,8 +23,6 @@ data class TaakConfig(
     var properties: TaakConfigProperties = TaakConfigProperties(),
 ) {
     data class TaakConfigProperties(
-        @Deprecated("Use version 2")
-        var typeUrl: String = "",
         var typeUrlV2: String = "",
```

Comment:
Do you think we could rename this property now to something more verbose? maybe `objectTypeUrl`?
I was already asked by a user what the "type" in here means. And I feel like keeping the V2 in there could make people think there is also a V1 option.

Reply:
Yess, I am aware. This is in a major release so I feel like this is justified and it reduces confusion.